Source/SwiftLintFramework/Rules/RuleConfigurations/VerticalWhitespaceClosingBracesConfiguration.swift

```diff
@@ -0,0 +1,38 @@
+private enum ConfigurationKey: String {
+    case severity = "severity"
+    case onlyEnforceBeforeTrivialLines = "only_enforce_before_trivial_lines"
+}
+
+// swiftlint:disable:next type_name
```

Comment:
Instead of going for the long type name here, why not nest it in the `VerticalWhitespaceClosingBracesRule` type? E.g. `VerticalWhitespaceClosingBracesRule.Configuration`

Reply:
Done.

Reply:
Note that I moved the definition of the configuration from its own file into the `VerticalWhitespaceClosingBracesRule.swift` file, to avoid having to ignore additional rules (the filename and the extension of `VerticalWhitespaceClosingBracesRule` have different names).